CONTRIBUTING.md

```diff
@@ -62,6 +62,7 @@ In this repo:
 In a second local clone of this repo in a different folder:
 
 - make sure you're on the gh-pages branch
+- run `npm install`
```

Comment:
This is in the docs instructions, and you need to run all of those including bundler. Maybe we should change this line to "Run the docs installation" and link to the install section? https://github.com/cfpb/capital-framework/tree/gh-pages#installation